interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -238,7 +238,13 @@ namespace cling {
 
       Expr* call = m_Sema.ActOnCallExpr(S, unresolvedLookup, noLoc,
                                         args, noLoc).get();
-      return call;
+
+      SourceLocation loc;
+      TypeSourceInfo* TSI
+              = m_Context.getTrivialTypeSourceInfo(Arg->getType(), noLoc);
```

Comment:
Can we maybe use the loc of the original pointer expression? Maybe also for the call expression itself?